src/main/resources/application-custom.yaml

```diff
@@ -15,7 +15,7 @@ spring:
     check-location: false
     baselineOnMigrate: true
   datasource:
-    url: ${datasource.url}
+    url: jdbc:mysql://${DB_HOST}:${DB_PORT}/${DB_NAME}?${DB_PARAMS}
```

Comment:
Are all task definitions using `jdbc:mysql://`?

Reply:
yes

Reply:
Do you thin we should allow for PostgresQL? I can easily add DB_PROTOCOL with default value of mysql